test/Microsoft.Azure.WebJobs.Host.FunctionalTests/TableTests.cs

```diff
@@ -143,7 +143,7 @@ public void Table_IfBoundToICollectorJObject_AddInsertsEntity()
             Assert.NotNull(entity.Properties);
 
             AssertPropertyValue(entity, "ValueStr", "abcdef");
-            AssertPropertyValue(entity, "ValueNum", 123);         
```

Comment:
BTW -- if it's hard to review b/c of all the spacing changes, you can add ?w=1 to the URL to ignore whitespace changes: https://github.com/Azure/azure-webjobs-sdk/pull/970/files?w=1